dotnet/src/webdriver/Remote/RemoteWebDriver.cs

```diff
@@ -172,7 +172,7 @@ public RemoteWebDriver(ICommandExecutor commandExecutor, ICapabilities desiredCa
                 // Only add the legacy log commands if the driver supports
                 // retrieving the logs via the extension end points.
                 this.CommandExecutor.CommandInfoRepository.TryAddCommand(DriverCommand.GetAvailableLogTypes, new CommandInfo(CommandInfo.GetCommand, "/session/{sessionId}/se/log/types"));
-                this.CommandExecutor.CommandInfoRepository.TryAddCommand(DriverCommand.GetLog, new CommandInfo(CommandInfo.GetCommand, "/session/{sessionId}/se/logs"));
+                this.CommandExecutor.CommandInfoRepository.TryAddCommand(DriverCommand.GetLog, new CommandInfo(CommandInfo.PostCommand, "/session/{sessionId}/se/logs"));
```

Comment:
As per https://github.com/SeleniumHQ/selenium/issues/7390#issuecomment-518429009 also URL should be probably changed to one without trailing "s" ("/session/{sessionId}/se/log")